ts/Grid/Pro/CellEditing/CellEditingComposition.ts

```diff
@@ -165,13 +165,25 @@ namespace CellEditingComposition {
             ](column, editModeOptions?.renderer || {});
         }
 
+        if (!CellRendererRegistry.types[staticRendererTypeName]) {
+            staticRendererTypeName = 'text';
+        }
+
         const staticRendererType = CellRendererRegistry.types[
             staticRendererTypeName
         ];
 
         let defRenderer = staticRendererType.defaultEditingRenderer;
         if (typeof defRenderer !== 'string') {
-            defRenderer = defRenderer[column.dataType];
+            let renderer = defRenderer[column.dataType];
+
+            if (!renderer) {
+                // eslint-disable-next-line no-console, max-len
+                console.warn(`The cell data type "${column.dataType}" is not registered. Using default string renderer instead.`);
```

Comment:
This warning is misleading - here we do not check if the cell data type is registered, but what is the default edit mode renderer for this data type in a given view/static renderer.

I don't know if it's necessary to throw a warning here at all - if someone hasn't defined a specific edit mode renderer for a given type, just render `textInput`.